src/main/java/org/apache/commons/beanutils2/converters/SqlTimeConverter.java

```diff
@@ -68,9 +68,7 @@ public SqlTimeConverter(final Object defaultValue) {
     }
 
     /**
-     * Gets a {@code DateFormat} for the Locale.
-     * @param locale TODO
-     * @param timeZone TODO
+     * {@inheritDoc}
```

Comment:
Hi. You removed some of the tags but not the return tag, why some and not that one?

Reply:
Valid 🤔 

I remember at the time thinking I'll leave the `@return` tag because it was different, so I assumed it should be overridden.
Now looking at it... I realize it's "different" but not actually different.

I'll remove it. ^-^'